app/static/js/console.UI.js

```diff
@@ -47,7 +47,7 @@ function addClusterListPanelJQEventListeners() {
     // Bootstrap carousel as clusters / cluster details / create cluster slider init
     $jq('.carousel').carousel('pause');
     // show cluster details
-    $jq(document).on("click", ".cluster h4 .btn-cluster", function() {
+    $jq(document).on("click", ".cluster h4 #btn-cluster", function() {
```

Comment:
Because in h4 there are two items with class btn-cluster, and only one needs the click event.